src/main/java/service/ValidatorServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package service;
+
+import java.util.List;
+
+public class ValidatorServiceImpl implements ValidatorService {
+
+    @Override
+    public void isValidate(List<String> list) {
+        if(list.size() == 0) {
+            throw new RuntimeException("File is Empty");
+        }
+        list.remove(0);
```

Comment:
What will be here if first line will not be like: "type,fruit,quantity". What you delete here?